### ISSUE TITLE
Compiling config to name the lib*.lib files with "lib*-3.lib"  

### ISSUE THREAD

**Issue by hlsantos** [openssl/openssl#20350]
Hi  Folks, I need to compile the library files libcrypto.lib and libssl.lib to differentiate between 1.1.1 and 3.0.   I compiled all versions  1.1.1.t and 3.0.8 now, and with my large setup, between debug/release, xp/non-xp, 32/64 bit, etc,  there are a few linking differences.  It would better to just have the properly named library so I can eliminate build issues.   At some point, we can just stick with OpenSSL v3 but not yet.  I need both, 1.1.1t for production and 3.0.8 for gamma testing it, which I found is problematic with TLS v1.1 compatibility (yes, I know the risk, deprecated, but if the options are available, I need to make it available i.e. possible to connect/negotiate proper ciphers, signature algorythms). 

Currently, I have a xp-support.config that is copied to the OpenSSL-x.x.x folder for the build process.

```
# 04/21/19 08:03 am, SSI/HLS
# Adds XP/2003 support
# usage from within build folder:
# perl ..\configure VC-WIN32-XP --config=..\xp-support.conf

my %targets = (
    "VC-WIN32-XP" => {
        inherit_from => [ "VC-WIN32" ],
        lflags       => '/subsystem:console,"5.01" /opt:ref',
        bin_lflags   => '/subsystem:console,"5.01" /opt:ref',

    },
    "VC-WIN64A-XP" => {
        inherit_from => [ "VC-WIN64A" ],
        lflags       => '/subsystem:console,"5.02" /opt:ref',
        bin_lflags   => '/subsystem:console,"5.02" /opt:ref',
    },
);

```
Would this be he proper place to add or should new rename-lib.config be created to set the /importlib:name?

Thanks

**Comment by mattcaswell** [comment]
@levitte?

**Comment by nhorman** [comment]
marking as inactive, will be closed at the completion of 3.4 dev without further input
